tex/tutorial.tex

```diff
@@ -3,19 +3,12 @@ \chapter{Language Usage Tutorial}
 This will cover the configuration of the user's environment and the usage of Extend's features.
 
 \section{Setup}
-The Extend compiler requires that the OCaml Language and LLVM be installed on the host machine. Development was done in a virtual machine running the 64-bit Ubuntu 16.04 operating system.
+The Extend compiler requires that the OCaml Language and LLVM be installed on the host machine. Development was done in a virtual machine running the 64-bit Ubuntu operating system. In order to quickly get Extend up and running, please use \underline{\href{https://courseworks2.columbia.edu/courses/10787/files/673708/download}{this virtual machine}}, which has been provided as part of the course.
```

Comment:
You could mention the .travis-ci.sh script. This is the install script on a bare bones machine.